src/material/badge/badge.ts

```diff
@@ -225,17 +218,17 @@ export class MatBadge extends _MatBadgeBase implements OnDestroy, OnChanges, Can
     return badgeElement;
   }
 
-  /** Sets the aria-label property on the element */
+  /** Sets the aria-describedby property on the host element */
   private _updateHostAriaDescription(newDescription: string, oldDescription: string): void {
     // ensure content available before setting label
-    const content = this._updateTextContent();
+    this._updateTextContent();
```

Comment:
nit: I think that we can also make it so that `_updateTextContent` doesn't return the badge element. This is the only place where the return value is used.